jobs/kola-gcp.Jenkinsfile

```
@@ -103,7 +104,7 @@ cosaPod(memory: "512Mi", kvm: false,
                         platformArgs: """-p=gcp \
                             --gcp-json-key=\${GCP_KOLA_TESTS_CONFIG} \
                             --gcp-project=${gcp_project} \
-                            --gcp-confidential-vm""")
+                            --gcp-confidential-type sev-snp""")
```

Let's try not to rely on the undocumented functionality up front and go with the documented version:

```suggestion
                            --gcp-confidential-type sev_snp""")
```

---

SGTM, updated, thanks!